adapter/config/default_config.go

```diff
@@ -121,6 +121,20 @@ var defaultConfig = &Config{
 				IdleTimeoutInSeconds:           3600,
 			},
 		},
+		RateLimit: rateLimit{
+			Enabled:  false,
+			Hostname: "ratelimit-service",
+			Port:     8083,
+			FailureModeDeny: false,
+			RatelimitRequestTimeouts: ratelimitRequestTimeouts{
+				RequestTimeoutInNanoSeconds: 80000000,
```

Comment:
Is this the value we agreed upon @renuka-fernando

Reply:
Yes, 80ms